package-lock.json

```diff
@@ -5,7 +5,7 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "2.9.0",
+      "version": "2.10.0",
```

Comment:
Should be already on 2.10.0 right?

Reply:
Apparently not https://github.com/aws/amazon-chime-sdk-js/blob/master/package-lock.json#L8